src/useCircularDrag.ts

```diff
@@ -25,10 +29,16 @@ export function useCircularDrag(ref: RefObject<SVGElement | null>) {
 		[onChange, getValueFromPointerEvent]
 	)
 
-	const handleEnd: EventListener = (e) => {
-		stopEvent(e)
-		setDragging(false)
-	}
+	const handleEnd: EventListener = useCallback(
+		(e) => {
+			if (!onChangeEnd) return
```

Comment:
I believe this should be 2 lines below to allow the event to be reliably stopped and dragging to be set to false?

Reply:
Yes you are right :)